python/xir/ir.lark

```diff
@@ -58,6 +61,8 @@ _GATE: "gate"
 _FUNC: "func"
 _OP: "operator"
 _OUT: "output"
+_ADJOINT: "adjoint"
```

Comment:
Do we prefer `adjoint` or `adj` or some other shorter version? Probably `adjoint` is fine, but just wanted to raise the question.

Reply:
Or potentially `inv`? I just realize that this is even more general, e.g., in the case that the operation to be applied isn't unitary?

Reply:
I'm have no preference really. I think either `adj`, `adjoint` or `inv` sounds good. Maybe `inv` is nice, since it can be used more generally, as @josh146 mentioned. 🤔

Reply:
I would prefer to use `adjoint` unless we intend to support non-unitary gates (i.e., the tighter the bound on the modifier, the better); however, I will concede that `inv` has the advantage of also being used in OpenQASM 3.

Reply:
Note that in PennyLane we do support non-unitary operations (even though we use `qml.adjoint`, lol). @trbromley, thoughts?